src/doc.d

```diff
@@ -345,7 +345,7 @@ extern (C++) static TemplateDeclaration getEponymousParent(Dsymbol s)
ult = "DDOC =  <html><head>\n        <META http-equiv=\"content-type\" content=\"text/html; charset=utf-8\">\n        <title>$(TITLE)</title>\n        </head><body>\n        <h1>$(TITLE)</h1>\n        $(BODY)\n        <hr>$(SMALL Page generated by $(LINK2 http://dlang.org/ddoc.html, Ddoc). $(COPYRIGHT))\n        </body></html>\n\nB =     <b>$0</b>\nI =     <i>$0</i>\nU =     <u>$0</u>\nP =     <p>$0</p>\nDL =    <dl>$0</dl>\nDT =    <dt>$0</dt>\nDD =    <dd>$0</dd>\nTABLE = <table>$0</table>\nTR =    <tr>$0</tr>\nTH =    <th>$0</th>\nTD =    <td>$0</td>\nOL =    <ol>$0</ol>\nUL =    <ul>$0</ul>\nLI =    <li>$0</li>\nBIG =   <big>$0</big>\nSMALL = <small>$0</small>\nBR =    <br>\nLINK =  <a href=\"$0\">$0</a>\nLINK2 = <a href=\"$1\">$+</a>\nLPAREN= (\nRPAREN= )\nBACKTICK= `\nDOLLAR= $\nDEPRECATED= $0\n\nRED =   <font color=red>$0</font>\nBLUE =  <font color=blue>$0</font>\nGREEN = <font color=green>$0</font>\nYELLOW =<font color=yellow>$0</font>\nBLACK = <font color=black>$0</font>\nWHITE = <font color=white>$0</font>\n\nD_CODE = <pre class=\"d_code\">$0</pre>\nDDOC_BACKQUOTED = $(D_INLINECODE $0)\nD_INLINECODE = <pre style=\"display:inline;\" class=\"d_inline_code\">$0</pre>\nD_COMMENT = $(GREEN $0)\nD_STRING  = $(RED $0)\nD_KEYWORD = $(BLUE $0)\nD_PSYMBOL = $(U $0)\nD_PARAM   = $(I $0)\n\nDDOC_COMMENT   = <!-- $0 -->\nDDOC_DECL      = $(DT $(BIG $0))\nDDOC_DECL_DD   = $(DD $0)\nDDOC_DITTO     = $(BR)$0\nDDOC_SECTIONS  = $0\nDDOC_SUMMARY   = $0$(BR)$(BR)\nDDOC_DESCRIPTION = $0$(BR)$(BR)\nDDOC_AUTHORS   = $(B Authors:)$(BR)\n$0$(BR)$(BR)\nDDOC_BUGS      = $(RED BUGS:)$(BR)\n$0$(BR)$(BR)\nDDOC_COPYRIGHT = $(B Copyright:)$(BR)\n$0$(BR)$(BR)\nDDOC_DATE      = $(B Date:)$(BR)\n$0$(BR)$(BR)\nDDOC_DEPRECATED = $(RED Deprecated:)$(BR)\n$0$(BR)$(BR)\nDDOC_EXAMPLES  = $(B Examples:)$(BR)\n$0$(BR)$(BR)\nDDOC_HISTORY   = $(B History:)$(BR)\n$0$(BR)$(BR)\nDDOC_LICENSE   = $(B License:)$(BR)\n$0$(BR)$(BR)\nDDOC_RETURNS   = $(B Returns:)$(BR)\n$0$(BR)$(BR)\nDDOC_SEE_ALSO  = $(B See Also:)$(BR)\n$0$(BR)$(BR)\nDDOC_STANDARDS = $(B Standards:)$(BR)\n$0$(BR)$(BR)\nDDOC_THROWS    = $(B Throws:)$(BR)\n$0$(BR)$(BR)\nDDOC_VERSION   = $(B Version:)$(BR)\n$0$(BR)$(BR)\nDDOC_SECTION_H = $(B $0)$(BR)\nDDOC_SECTION   = $0$(BR)$(BR)\nDDOC_MEMBERS   = $(DL $0)\nDDOC_MODULE_MEMBERS = $(DDOC_MEMBERS $0)\nDDOC_CLASS_MEMBERS  = $(DDOC_MEMBERS $0)\nDDOC_STRUCT_MEMBERS = $(DDOC_MEMBERS $0)\nDDOC_ENUM_MEMBERS   = $(DDOC_MEMBERS $0)\nDDOC_TEMPLATE_MEMBERS = $(DDOC_MEMBERS $0)\nDDOC_ENUM_BASETYPE = $0\nDDOC_PARAMS    = $(B Params:)$(BR)\n$(TABLE $0)$(BR)\nDDOC_PARAM_ROW = $(TR $0)\nDDOC_PARAM_ID  = $(TD $0)\nDDOC_PARAM_DESC = $(TD $0)\nDDOC_BLANKLINE  = $(BR)$(BR)\n\nDDOC_ANCHOR     = <a name=\"$1\"></a>\nDDOC_PSYMBOL    = $(U $0)\nDDOC_PSUPER_SYMBOL = $(U $0)\nDDOC_KEYWORD    = $(B $0)\nDDOC_PARAM      = $(I $0)\nDDOC_CONSTRAINT      = $(BR)<span class=\"constraint\">if($0)</span>\nDDOC_OVERLOAD_SEPARATOR      = $(BR)\n\nESCAPES = /</&lt;/\n          />/&gt;/\n          /&/&amp;/\n";
```

Comment:
Since you touched this anyway, any chance to take advantage of D and convert it to a civilized string? E.g. use actual newlines etc. Thanks!